packages/scanner/test/cli/scan.spec.ts

```diff
@@ -288,6 +289,17 @@ describe('scan', () => {
       await waitForSingleFinding();
     });
 
+    it('does not raise on EACCES: permission denied', async () => {
+      await createIndex(secretInLogMap);
+      const permissionDeniedDir = join(tmpDir, 'permission_denied_dir');
+      await mkdir(permissionDeniedDir);
+      await fsextra.copy(secretInLogMap, join(permissionDeniedDir, basename(secretInLogMap)));
+      chmod(permissionDeniedDir, 0o000);
+      await createWatcher();
+      await waitForSingleFinding();
+      chmod(permissionDeniedDir, 0o777); // else the next testcase fails
```

Comment:
This should (?) be deterministic but it'd be good to `await` these `chmod` calls in case something changes in the future.